src/mixins/PropertyMixin.js

```diff
@@ -21,7 +21,7 @@
  */
 import debounce from 'debounce'
 import Contact from '../models/contact'
-import ICAL from 'ical.js'
+import setPropertyAlias from '../services/updateDesignSet'
```

Comment:
Hum, this would be the default export, not the function

Reply:
Augh. That sneaky Quality Review thingy previously flagged it as an error, so I just believed it... https://app.codacy.com/manual/skjnldsv/contacts/commit?cid=430033681&fileBranchId=15882771#newIssuesView

Reply:
DOn't listent too much to this tool, it's not very great to be honest :/